### ISSUE TITLE
Is the model trained on multi-task learning?

### ISSUE THREAD

**Issue by sj584** [mini-ops996/VirulentHunter#3]
First of all. thank you for your work.
I'm considering using your dataset for model training and validation.

However, I pose some questions regarding multi-task learning.
From the figure in the paper, it does look like multi-task learning while the code and data seems to say opposite.
Here are the observations I got from your work.

1. The paper does not show any benefits from multi-task learning.

2. There is an overlap gap between the binary training set and multi-class training set  
->(also, there is no test set for the multi-class dataset, why is that?)

3. The code you provided for training seems to train the model separately 

4. The code you provided for inference also predict the task separately

In general, I think your work is great, in either way (multi-task or not). 
I just wanted to understand more about your study.

Thank you in advance for your reply :)

**Comment by sj584** [comment]
Another thing,

It seems that dataset contains protein sequences upto the length of 2000.
How did you use ESM-2 model for the length over 1022?

I wonder whether you trimmed it or not

Thanks
